lib/main.dart

```diff
@@ -45,6 +46,7 @@ ThemeData appTheme() {
 
     textTheme: TextTheme(
       headline: TextStyle(fontSize: 30, color: Colors.white),
+      body1: TextStyle(fontSize: 16, color: Colors.white),
```

Comment:
Since we are setting the font size in the body widget we don't need to do so here

Reply:
I will change this for you, did you find out what caused the underlines when I removed the body1 theme if we can fix that I don't see why we can't remove the theme again

Reply:
I have pushed changes for this

Reply:
> I will change this for you, did you find out what caused the underlines when I removed the body1 theme if we can fix that I don't see why we can't remove the theme again

It seems to be related to not using a theme

Reply:
ok well we will just have to leave the color there then maybe raise an issue on the flutter repo